omnigan/losses.py

```diff
@@ -395,6 +418,7 @@ def get_losses(opts, verbose, device=None):
     # task losses
     # ? * add discriminator and gan loss to these task when no ground truth
     # ?   instead of noisy label
+    
```

Comment:
To be consistent with the rest of the code, maybe we should take the scaler out of the loss and put it in trainer.py? Like we did for TV Loss today.